pandas/tests/io/parser/common/test_file_buffer_url.py

```diff
@@ -330,8 +331,9 @@ def test_read_csv_file_handle(all_parsers, io_class, encoding):
     parser = all_parsers
     expected = DataFrame({"a": [1], "b": [2]})
 
-    content = "a,b\n1,2"
+    content: Union[str, bytes] = "a,b\n1,2"
```

Comment:
```suggestion
    content: str | bytes = "a,b\n1,2"
```

:grin:

we have 
```
[mypy-pandas.tests.*]
check_untyped_defs=False
```
so not sure why mypy is checking this function anyway.

OTOH could refactor...

```
    content = "a,b\n1,2"
    handle = io_class(content.encode("utf-8") if io_class == BytesIO else content)
```

Reply:
Hm the refactor is better. Thx.